src/main/scala/io/iohk/ethereum/network/package.scala

```diff
@@ -19,4 +25,26 @@ package object network {
       Hex.decode(nodeId)
     curve.getCurve.decodePoint(bytes)
   }
+
+  def loadAsymmetricCipherKeyPair(filePath: String): AsymmetricCipherKeyPair = {
+    if(!new File(filePath).exists()){
+      val keysValuePair = generateKeyPair()
+
+      //Write keys to file
+      val (pub, priv) = AsymmetricCipherKeyPairSerializable.toHexStrings(keysValuePair)
+      val file = new PrintWriter(filePath)
+      try {
+        val keyFileFormat: String = System.getProperty("key-file-format")
+        file.write(keyFileFormat.format(pub, priv))
+      } finally {
+        file.close()
+      }
+
+      keysValuePair
+    } else {
+      val List(pub, priv) = Source.fromFile(filePath).getLines().toList
```

Comment:
what will happen if I change file format from `-Dkey-file-format=%s\n%s` for example to -Dkey-file-format=%s %s